habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/exec/PoetryCommandHelper.java

```diff
@@ -133,6 +133,26 @@ public int executeAndLogOutput(List<String> arguments) throws MojoExecutionExcep
         return executor.executeAndRedirectOutput(logger);
     }
 
```

Comment:
Overloaded `executeAndLogOutput()` to include `Map<String, String> environmentVariables` parameter